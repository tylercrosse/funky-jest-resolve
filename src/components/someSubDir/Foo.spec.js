import React from 'react';
import renderer from 'react-test-renderer';
import Foo from './Foo';

describe('<Foo />', () => {
  it('should render correctly', () => {
    const renderedComponent = renderer.create(<Foo />);
    const tree = renderedComponent.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
