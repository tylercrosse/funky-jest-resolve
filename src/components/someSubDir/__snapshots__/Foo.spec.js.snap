exports[`<Foo /> should render correctly 1`] = `
<h1>
  Foo
</h1>
`;
